test/Interop/Cxx/class/Inputs/constructors.h

```diff
@@ -337,6 +337,34 @@ __attribute__((swift_attr("release:Release14"))) ParameterizedCtor {
   int val = 1;
   ParameterizedCtor(int x) : val(x) {}
 };
+
+struct __attribute__((swift_attr("import_reference")))
+__attribute__((swift_attr("retain:Retain16")))
+__attribute__((swift_attr("release:Release16"))) ParameterizedCtor2 {
+public:
+  int val1 = 1;
+  int val2 = 1;
+  ParameterizedCtor2() {};
+  ParameterizedCtor2(int x) : val1(x) {}
+  ParameterizedCtor2(int x, int y) : val1(x), val2(y) {}
+};
+
+struct __attribute__((swift_attr("import_reference")))
+__attribute__((swift_attr("retain:Retain17")))
+__attribute__((swift_attr("release:Release17"))) VariadicCtors {
+public:
+  int val1 = 1;
+  int val2 = 2;
+  VariadicCtors(...) {};
+};
+
+struct __attribute__((swift_attr("import_reference")))
+__attribute__((swift_attr("retain:Retain18")))
+__attribute__((swift_attr("release:Release18"))) NoIdentifierInCtorParam {
+public:
```

Comment:
Would be nice to have some tests with constructors of user defined types that have user defined copy/move ctors.